mlrun/__main__.py

```diff
@@ -907,6 +907,12 @@ def logs(uid, project, offset, db, watch):
     "https://apscheduler.readthedocs.io/en/3.x/modules/triggers/cron.html#module-apscheduler.triggers.cron."
     "For using the pre-defined workflow's schedule, set --schedule 'true'",
 )
+@click.option(
+    "--overwrite",
+    "-o",
```

Comment:
general suggestion feel free to ignore it.
what if we call the flag `--overwrite-schedule` and `-os`? just to make it a bit clearer what the user is overwriting.

Reply:
Agreed, let's be specific with the naming of the argument

Reply:
np